lib/search/models/search_results.dart

```diff
@@ -187,9 +202,36 @@ class SearchQuery {
     required this.value,
   });
 
-  String get withoutSort => value.split(':').sublist(2).join(':');
+  String get searchTerm {
+    return value.split(':')[0];
+  }
+
+  String get sort {
+    return value.split(':')[1];
+  }
+
+  Map<String, String> get filters {
+    final parts = value.split(':');
+    var filters = <String, String>{};
+
+    if (parts.length == 2) {
+      // Just the sort. Example: `cybershot:relevance` or `:relevance`
+      return {};
+    }
```

Comment:
Unused for now, but I think it is useful for understanding the meaning of the hybris query.

Moreover I feel we can fix some issues in the Sort/Filter page (ex: the filters don't appear as selected IIRC)